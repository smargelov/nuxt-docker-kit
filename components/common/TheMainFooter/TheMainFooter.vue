<template lang="pug">
  footer.b-footer
    p._test FOOTER
</template>

<script>
export default {
  name: "TheMainFooter",
  props: {},
  data() {
    return {};
  },
  computed: {}
};
</script>

<style lang="sass">
@import './TheMainFooter'
</style>
