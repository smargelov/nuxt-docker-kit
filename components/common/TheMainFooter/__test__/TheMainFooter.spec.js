import { mount } from "@vue/test-utils"
import TheMainFooter from "../TheMainFooter"

describe("TheMainFooter", () => {
  const wrapper = mount(TheMainFooter)
  it("render correctly", () => {
    expect(wrapper.vm.$el).toMatchSnapshot()
  })
  test('is a Vue instance', () => {
    expect(wrapper.vm).toBeTruthy()
  })
})
