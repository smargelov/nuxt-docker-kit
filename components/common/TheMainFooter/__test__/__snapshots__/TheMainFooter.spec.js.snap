// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheMainFooter render correctly 1`] = `
<footer
  class="b-footer"
>
  <p
    class="_test"
  >
    FOOTER
  </p>
</footer>
`;
